From 6cbb2fe4908464e78fdc5f714d66eda9cac9b19f Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
Date: Thu, 29 Feb 2024 09:40:50 +0100
Subject: [PATCH 0922/1085] media: ov64a40: Align '=' to tab

Small cosmetic to align the '=' signs to the next tab in the i2c
driver structure declaration declaration.

Signed-off-by: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
---
 drivers/media/i2c/ov64a40.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/i2c/ov64a40.c
+++ b/drivers/media/i2c/ov64a40.c
@@ -3675,7 +3675,7 @@ static struct i2c_driver ov64a40_i2c_dri
 		.of_match_table	= ov64a40_of_ids,
 		.pm = &ov64a40_pm_ops,
 	},
-	.probe = ov64a40_probe,
+	.probe	= ov64a40_probe,
 	.remove	= ov64a40_remove,
 };
 
